python_modules/dagster/dagster/_core/asset_graph_view/asset_graph_view.py

```diff
@@ -289,16 +289,17 @@ def for_test(
         from dagster._core.definitions.data_version import CachingStaleStatusResolver
         from dagster._core.instance import DagsterInstance
 
+        instance = instance or DagsterInstance.ephemeral()
         stale_resolver = CachingStaleStatusResolver(
-            instance=instance or DagsterInstance.ephemeral(),
+            instance=instance,
             asset_graph=defs.get_asset_graph(),
         )
         check.invariant(stale_resolver.instance_queryer, "Ensure instance queryer is constructed")
         return AssetGraphView(
             stale_resolver=stale_resolver,
             temporal_context=TemporalContext(
                 effective_dt=effective_dt or pendulum.now(),
-                last_event_id=last_event_id,
+                last_event_id=last_event_id or instance.event_log_storage.get_maximum_record_id(),
```

Comment:
it would be cool to have this state be known / logged somewhere so there is observability in to the "I am starting fresh tracking from this time forward" state versus normal operation delta from last cursor / check